keras_v2.py

```diff
@@ -0,0 +1,116 @@
+import os
+import numpy as np
+import keras
+from keras.applications import ResNet50
+from keras.callbacks import EarlyStopping, ModelCheckpoint
+from keras.preprocessing.image import ImageDataGenerator
+from keras import optimizers
+from keras.models import Sequential, Model
+from keras.layers import Dropout, Flatten, Dense
+
+np.random.seed(1337) # for reproducibility
+
+os.environ["CUDA_VISIBLE_DEVICES"] = "0" # only use first GPU
+
+def get_callbacks(weights_path, patience=30, monitor='val_loss'):
```

Comment:
we can make a `callback.py` and store our callbacks there to make it modular.

Reply:
oh I think you want to just `python keras_v2.py` so maybe its best to store everything together in 1 file for newbies to understand.